yb-voyager/cmd/root.go

```diff
@@ -63,6 +63,13 @@ var rootCmd = &cobra.Command{
 Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like setting up source/target, migration workflow etc.`,
 
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		// Initialize the config file
+		err := initConfig(cmd)
+		if err != nil {
+			// not using utils.ErrExit as logging is not initialized yet
+			fmt.Printf("ERROR: Failed to initialize config: %v\n", err)
+			atexit.Exit(0)
```

Comment:
we shouldn't exit with 0. 0 is indicative of success. atexit.Exit(1) should be okay.